prep-css-syntax/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<style>
+
+
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>CSS Syntax</title>
```

Comment:
The `<meta>` tags should be outside of the `<style>` tags. Only CSS rules go inside of `<style>` tags.